src/application/TaskData.java

```diff
@@ -67,8 +89,40 @@ public String getDescription() {
 		return description;
 	}
 	
-	public String getTaskID() {
-		return taskID;
+	public String toString() {
+		String str = "";
+		String.format("%20s", "TaskType: ");
+		str = str + taskType + "\n" + description + "\n" + startDateTime + "\n" + endDateTime + "\n" + deadline + "\n" + status + "\n";
+		return str;
+	}
+	
```

Comment:
are u implying TaskData is directly talking to UI? not through any other components?
i prefer to return a String, String can contain "\n" for new lines.